src/media/css/buttons.styl

```diff
@@ -163,6 +164,18 @@ a.button {
         position: relative;
         vertical-align: bottom;
         width: fit-content;
+
+        +website-detail() {
```

Comment:
This resolves to [data-page-type~="detail"][data-page-type~="website] .mkt-app-button em which doesn't seem like it'll be used